example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -398,15 +398,26 @@ describe('The S3 Ingest Granules workflow', () => {
       expect(resourceURLs.includes(s3CredsUrl)).toBe(true);
     });
 
-    it('updates the CMR metadata "online resources" with the proper types', () => {
+    it('updates the CMR metadata "online resources" with the proper types and urls', () => {
       const resource = ummCmrResource;
-      const expected = [
+      const distributionUrl = getDistributionFileUrl({
+        bucket: files[0].bucket,
+        key: files[0].filepath
+      });
+      const s3Url = getPublicS3FileUrl({ bucket: files[2].bucket, key: files[2].filepath });
+      const s3CredsUrl = resolve(process.env.DISTRIBUTION_ENDPOINT, 's3credentials');
+      const expectedTypes = [
         'GET DATA',
         'VIEW RELATED INFORMATION',
         'VIEW RELATED INFORMATION',
         'GET RELATED VISUALIZATION'
       ];
-      expect(expected).toEqual(resource.map((r) => r.Type));
+      const cmrUrls = resource.map((r) => r.URL);
+
+      expect(cmrUrls.includes(distributionUrl)).toBe(true);
+      expect(cmrUrls.includes(s3Url)).toBe(true);
+      expect(cmrUrls.includes(s3CredsUrl)).toBe(true);
+      expect(expectedTypes).toEqual(resource.map((r) => r.Type));
```

Comment:
@Jkovarik Since the IngestGranule payload updated the `.jpg` file to be handled as "browse": 

https://github.com/nasa/cumulus/pull/867/files#diff-060cbca2e5356d00c14685b1c14b534eR38

To complete the AC of testing for "ancillary" URLs in CMR, should we be checking for this file as well?

Reply:
We are checking for this jpg file. 
In the IngestUMMSuccessSpec https://github.com/nasa/cumulus/blob/master/example/spec/ingestGranule/IngestUMMGSuccessSpec.js#L312

And in this files the public jpg URL is being tested on line 418.

Reply:
Ah, thank for your clarifying. Could we possible update `s3Url` to `s3BrowseImageUrl` or something? I feel like that would make this somewhat clearer for the future. If so, could we do the same thing for line 397 above and in the `IngestUMMGSuccessSpec.js`?